anesthetic/samples.py

```diff
@@ -20,22 +21,19 @@
 from anesthetic.weighted_pandas import WeightedDataFrame, WeightedSeries
 
 
-class MCMCSamples(WeightedDataFrame):
-    """Storage and plotting tools for MCMC samples.
+class Samples(WeightedDataFrame):
+    """Storage and plotting tools for general samples.
 
     Extends the pandas.DataFrame by providing plotting methods and
     standardising sample storage.
```

Comment:
Could we now add a bit more detail about exactly what it extends, e.g. the logL and tex functions (no need to document limits in light of the upcoming #200 upgrades)

Reply:
Isn't that essentially covered by the corresponding kwargs as listed under `Parameters`?

Reply:
Agreed.